app/controllers/likes_controller.rb

```diff
@@ -19,9 +19,18 @@ def save_like
     end
   end
 
+  def new
+    @like.save
+  end
+
+  def delete
+    @like.destroy
```

Comment:
I think I want to destroy the existing_like? I was wondering how I can delete the record inside the save_like method? Simply call the delete() or there's any better way?

Reply:
@muazzamnashat 
This is how you would call `delete` in a rails form
https://stackoverflow.com/questions/7704973/getting-a-form-to-use-method-delete-rails

Reply:
@muazzamnashat 
In general, you want to organize your controller in this manner (see the controller in this example)
You don't "need" to do it this way but the rails community is more opinionated than others and will think what you are doing is odd.  There are reasons why you don't want to use the same method to do too many things but that will take a bit to explain.
https://www.tutorialspoint.com/ruby-on-rails/rails-scaffolding.htm